tendrl/monitoring_integration/sync/gluster_cluster_details.py

```diff
@@ -112,7 +112,7 @@ def get_volumes_details(cluster_key):
             deleted = etcd_utils.read(
                 cluster_key + "/Volumes/" + str(volume_id) + "/" + "deleted"
             ).value
-            if str(deleted).lower() == "false":
+            if str(deleted).lower() != "true":
```

Comment:
Volumes delete flag default value is None only, So we can't check flag is false

Reply:
In case default None str(deleted) would work?

Reply:
In case default None str(deleted) would work?

Reply:
yes it will work, i tested this

Reply:
str(None) is 'None' only, it won't raise any error

Reply:
ok